iceoryx2-pal/posix/build.rs

```diff
@@ -25,8 +28,17 @@ fn main() {
     println!("cargo:rustc-link-lib=acl");
     println!("cargo:rerun-if-changed=src/c/posix.h");
 
+    if !cc::Build::new().get_compiler().is_like_clang() {
+        println!("cargo:warning=Clang is not available, you may set up environment {} to avoid missing some headers.", ICEORYX2_CLANG_ARG);
+    }
+
+    let mut builder = bindgen::Builder::default();
+    if let Ok(value) = env::var(ICEORYX2_CLANG_ARG) {
```

Comment:
@elfenpiff @orecham I think we did not yet specify whether we want to use the `ICEORYX2` or the `IOX2` prefix for env variables. What is your opinion on this?

Reply:
done, rename the env var to `IOX2_CLANG_ARG`

Reply:
Hi @elBoberido @orecham could you kindly let me know if you have the other concern?